test/test_suite.py

```diff
@@ -320,3 +322,33 @@ def test_pass_uri_query_as_kwargs(self):
         )
         assert kwargs["encrypt"] == "true"
         assert kwargs["compress"] == "true"
+
+
+class  HANACompileTest(fixtures.TestBase, AssertsCompiledSQL):
+
+    __dialect__ = testing.db.dialect
+
+    @testing.only_on('hana')
+    @testing.only_if('hana+hdbcli')
```

Comment:
```suggestion
```

As the test is SQL specific, it should also work other HANA client libraries.

Reply:
It should highlight that this test is only valid/useful for the HANA dialect but it is probably not required.